### PR TITLE
[A11y] Tables com contact

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue #.

### Summary of Changes
To be accessible a table must have a header. 
See for examle this checklist: https://dequeuniversity.com/checklists/web/tables

Joomla provides a param show_headings for frontend tables. This removes the table header. 

This PR sets the table header visually-hidden so it is still present for assistive technologies. 

I open some similar PRs to make testing easier. 

### Testing Instructions
You need at least one contact
Make Menu Items for featured contacts and contact category
Set param table_headings in tab "list layouts" to "hide"
See the view and sourcecode ot the tables in frontend 

### Actual result BEFORE applying this Pull Request
There is no table heading visible, also in source code.

### Expected result AFTER applying this Pull Request
There is no table heading visible, but it is in sourcecode with class="visually-hidden"
Make sure that the table heading is shown when table_headings is set to yes, as before.

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 1aea3fe2f96adc32dbcfa767225dcc1c1e67d722<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40139">issues.joomla.org/tracker/joomla-cms/40139</a>.</sub>

**Comment by ChristineWk** [comment]
Before Patch - source code:
` <caption class="visually-hidden">`
After Patch:
` <caption class="visually-hidden">
                Contacts,
            </caption>
                <thead class="visually-hidden">`

File: components/com_contact/tmpl/featured/default_items.php [hunk 1]
```diff
@@ -64,8 +64,7 @@ class="inputbox" onchange="document.adminForm.submit();"
             <caption class="visually-hidden">
                 <?php echo Text::_('COM_CONTACT_TABLE_CAPTION'); ?>,
             </caption>
-            <?php if ($this->params->get('show_headings')) : ?>
-                <thead>
+                <thead<?php echo $this->params->get('show_headings', '1') ?:  ' class="visually-hidden"'; ?>>
```
Comment (Quy) [inline components/com_contact/tmpl/featured/default_items.php]:
```suggestion
                <thead<?php echo $this->params->get('show_headings', '1') ? '' : ' class="visually-hidden"'; ?>>
```

File: components/com_contact/tmpl/category/default_items.php [hunk 2]
```diff
@@ -85,23 +85,21 @@ class="inputbox" onchange="document.adminForm.submit();"
                 <caption class="visually-hidden">
                     <?php echo Text::_('COM_CONTACT_TABLE_CAPTION'); ?>,
                 </caption>
-                <?php if ($this->params->get('show_headings')) : ?>
-                    <thead>
-                        <tr>
-                            <th scope="col" id="categorylist_header_title">
-                                <?php echo HTMLHelper::_('grid.sort', 'JGLOBAL_TITLE', 'a.name', $listDirn, $listOrder, null, 'asc', '', 'adminForm'); ?>
-                            </th>
+                <thead<?php echo $this->params->get('show_headings', '1') ? '' :  ' class="visually-hidden"'; ?>>
```
Comment (Quy) [inline components/com_contact/tmpl/category/default_items.php]:
```suggestion
                <thead<?php echo $this->params->get('show_headings', '1') ? '' : ' class="visually-hidden"'; ?>>
```

**Comment by chmst** [comment]
Thanks for fixes @Quy

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40139">issues.joomla.org/tracker/joomla-cms/40139</a>.</sub>

**Comment by obuisard** [comment]
Thanks again, Christiane @chmst!

**Pull request merged**